### PR TITLE
Add isRadio() and isTv() methods to programme

### PR DESCRIPTION
**Pull request by BjornTwachtmann**
Any objections? Please say there are no objections.

### CODE REVIEW

File: src/Domain/Entity/Programme.php [hunk 1]
```diff
@@ -281,6 +281,24 @@ public function getTleo(): Programme
         return $this;
     }
 
+    public function isRadio(): bool
+    {
+        $network = $this->getNetwork();
+        if ($network && $network->isRadio()) {
```
Comment (votemike) [inline src/Domain/Entity/Programme.php]: Or even `return $network && $network->isRadio()`?

File: src/Domain/Entity/Programme.php [hunk 2]
```diff
@@ -281,6 +281,24 @@ public function getTleo(): Programme
...
+            return true;
+        }
+        return false;
+    }
+
+    public function isTv(): bool
+    {
+        $network = $this->getNetwork();
+        if ($network && $network->isTv()) {
```
Comment (votemike) [inline src/Domain/Entity/Programme.php]: Or even `return $network && $network-> isTv()`?

**Comment by votemike** [comment]
Do we need tests for this change?

**Comment by BjornTwachtmann** [comment]
Goddammit I said no objections...

**Comment by votemike** [comment]
If you want no objections... Do-It-Right-In-The-First-Place™ 😝

**Comment by votemike** [comment]
Please create a release tag once merged.

**Pull request merged**